.github/scripts/muffet.sh

```diff
@@ -46,4 +46,8 @@ muffet http://localhost:1313 \
   --exclude "https://docs.google.com/document/d/14AuJ7SerLuOPESBjQlJqpBtzwSAoVf5ykTT7fjyJBT0/*" \
   --exclude "https://drive.google.com/file/d/1YPXoba9gVmD7SP-X88PpJIsIVGvY86_G.*" \
   --exclude "https://journals.sagepub.com/doi/full/10.1177/20552076231194924" \
+  --exclude "http[s]*://.*oppiamobile.readthedocs.io.*" \
+  --exclude "http[s]*://.*docs.couchdb.org.*" \
+  --exclude "http[s]*://.*borgbackup.readthedocs.io.*" \
+  --exclude "https://www.tableau.com" \
```

Comment:
Is this excluding these external sites from link checks? Why are these sites different from any others? Should we just exclude all external link checks? Listing out 30 regexes has a code smell...

Reply:
> I'm happy to do that, or send through the text, or review - whatever's helpful. 

I prefer you open a separate PR for updating the translation documentation so you won't be blocked by this one.

Reply:
Many websites return `403`, and as discussed [in Slack](https://medic.slack.com/archives/CUXB9SVT5/p1719250527341659?thread_ts=1719113183.530539&cid=CUXB9SVT5), this could be linked to some websites blocking our link checker agent,  or GitHub Runner IP being blocked by some shared infra like Cloudfare.

The list of excluded links is big, and we might need another approach to finding broken links. 

I nevertheless find the link checker helpful, as, for example, this weekend it caught POE Editors changes and Sonarqube doc link changes.